### PR TITLE
Add verify_listkeys_eqcfsm test.

### PR DESCRIPTION
**Pull request by bowrocker**
Test has already been merged into riak_test develop.

### CODE REVIEW

**Comment by bowrocker** [comment]
- Backends shouldn't matter
- We've been testing on 2.0, so 2.0+ if that's possible
- No upgrade parameter

File: db/seed.rb [hunk 1]
```diff
@@ -40,6 +40,7 @@
   verify_leave
   verify_link_walk_urls
   verify_listkeys
+  verify_listkeys_eqcfsm
```
Comment (seancribbs) [inline db/seed.rb]: Given your answers to the questions I asked, this should probably go down in the Riak 2.0 section.

**Comment by seancribbs** [comment]
:+1: merging and deploying

**Comment by seancribbs** [comment]
Deployed.

**Comment by bowrocker** [comment]
@seancribbs thank you!

**Pull request merged**